doc/charm++/manual.rst

```diff
@@ -10880,6 +10880,83 @@ The charm directory contains a collection of example-programs and
 test-programs. These may be deleted with no other effects. You may also
 ``strip`` all the binaries in ``charm/bin``.
 
+Installation for Specific Builds
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+UCX
+^^^
+UCX stands for Unified Communication X and is a high performance communication
+library that can be used as a backend networking layer for Charm++ builds on
+supported transports like Infiniband, Omni-Path, gni, TCP/IP, etc.
+
+In order to install Charm++ with UCX backend, you require UCX or HPC-X modules
+in your environment. In case UCX or HPC-X is not available in your environment,
+you can build UCX from scratch using the following steps:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/openucx/ucx.git
+   $ cd ucx
+   $ ./autogen.sh
+   $ ./contrib/configure-release --prefix=$HOME/ucx/build
+   $ make -j8
+   $ make install
+
+After installing UCX, there are several supported process management interfaces (PMI)
+that can be specified as options in order to build Charm++ with UCX. These include
+Simple PMI, Slurm PMI, Slurm PMI 2 and PMIx (using OpenMPI). Currently, in order to
+use PMIx for process management, it is required to have OpenMPI installed on the system.
+Additionally, in order to use the other supported process management interfaces, it is
+required to have a non-OpenMPI based MPI implementation installed on the system (e.g.
+Intel MPI, Mvapich, MPICH, etc.).
+
+The following section shows examples of build commands that can be used to build targets
+with the UCX backend using different process management interfaces. In all these cases,
+in order for Charm++ to find UCX path, it is required to pass the UCX build directory
+as ``--basedir``.
+
+To build the Charm++ target with Simple PMI, do not specify any additional option as shown
+in the build command.
+
+.. code-block:: bash
+
+   $ ./build Charm++ ucx-linux-x86_64 --with-production --enable-error-checking --basedir=$HOME/ucx/build -j16
```

Comment:
I think we usually keep charm++ in lower case in the build line.

Reply:
Good point, done.